src/BaseDAO/ShareTest/Common.hs

```diff
@@ -53,6 +54,14 @@ expectFailed
   => Address -> MText -> m () -> m ()
 expectFailed addr val = flip expectFailure (NettestFailedWith addr val)
 
+getTotalSupplyFromLedger :: Ledger -> TotalSupply
```

Comment:
Nitpick, but `get` seems to indicate some fetching, I suggest renaming this just to `totalSupplyFromLedger`